sdk/ml/azure-ai-ml/tests/schedule/unittests/test_schedule_entity.py

```diff
@@ -184,3 +184,11 @@ def test_invalid_date_string(self):
         with pytest.raises(Exception) as e:
             job_schedule._validate(raise_error=True)
         assert "Not a valid ISO8601-formatted datetime string" in str(e)
+    
+    ##findings question_answering is wrong task type which mention on https://github.com/Azure/azureml_run_specification/blob/momo-genai-spec/specs/momo-genai.md
```

Comment:
Is this comment referencing an issue with the spec itself? If so, let's forward this comment to Alex Hughes and remove it from this file.

The actual test looks good! Thanks for adding it.

Reply:
Yes the comment is about spec.
https://azuremlschemas.azureedge.net/latest/monitorSchedule.schema.json
![image](https://github.com/Azure/azure-sdk-for-python/assets/70937115/3a62931d-f477-4b4e-b455-267b7fdb02dc)